.github/workflows/releaseutils.yml

```diff
@@ -0,0 +1,25 @@
+name: Add sbom
+
+on:
+  workflow_dispatch:
+  workflow_run:
+        workflows: [test-latest-release]
+        types: [completed]
+
+jobs:
+    add-sbom:
+        name: Add SBOM
+        runs-on: ubuntu-latest
+        if: ${{github.event.workflow_run.conclusion=='success'}}
+        steps:
+            - name: checkout code
+              uses: actions/checkout@v4
+
+            - uses: advanced-security/sbom-generator-action@v0.0.1
+              id: sbom
+              env: 
+                    GITHUB_TOKEN: ${{ github.token }}

```

Comment:
Is there a way to give this Action only read access to the repo? That would be better.

Reply:
done

Reply:
currently gave read all permissions, but I think more granular access can be provided in the future

Reply:
I think read all is OK, since the repo already gives read all permission to everybody.